tests/integration/cgroups.bats

```diff
@@ -70,5 +70,11 @@ EOF
 
     # update kernel memory limit
     run "$RUNC" update test_cgroups_kmem --kernel-memory 50331648
-    [ ! "$status" -eq 0 ]
+    # Since kernel 4.6, we can update kernel memory without initialization
+    # because it's accounted by default.
+    if [ "$KERNEL_MAJOR" -lt 4 ] || [ "$KERNEL_MAJOR" -eq 4 -a "$KERNEL_MINOR" -le 5 ]; then
+        [ ! "$status" -eq 0 ]
+    else
+        [ "$status" -eq 0 ]
```

Comment:
Oh, sorry, I thought the second part read `[ "$KERNEL_MINOR" -eq 4 -a "$KERNEL_PATCH" -le 5 ]`. :P